include/net/coap.h

```diff
@@ -702,6 +702,9 @@ struct coap_pending *coap_pending_next_unused(
 struct coap_reply *coap_reply_next_unused(
 	struct coap_reply *replies, size_t len);
 
+void coap_pendings_clear(struct coap_pending *pendings, size_t len);
```

Comment:
Would be good to add doxygen docs for these functions, just as other coap functions have.